public/assets/css/main.css

```diff
@@ -1085,6 +1085,12 @@ h6,
     color: #262d3f;
     display: block;
     margin-bottom: 10px; }
+  .input-style-1 select{
+    width: 465px; 
+    height: 40px; 
+    border-radius: 5px;
+    border: 1px solid #4a6cf7;
+  }
```

Comment:
This bit isn't actually showing up for me

![image](https://github.com/user-attachments/assets/76818b04-8294-4745-8d80-d1379a5f58dd)


I think it is because you are styling the `select` whereas you need to style the `span` that the `select2` dropdown generates. Also it would be better if you could do the styles with bootstrap CSS classes. https://getbootstrap.com/docs/5.3/utilities/borders/#radius stuff line that.